packages/database/drizzle.config.ts

```
@@ -0,0 +1,19 @@
+import { config } from 'dotenv';
+import { defineConfig } from 'drizzle-kit';
+
+config({
+  path: ['.env.local', '.env'],
+});
+
+export default defineConfig({
+  out: './drizzle',
+  schema: './src/schemas/*.ts',
+  dialect: 'postgresql',
```

Biome (1.9.4)

[error] This property value named dialect is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named dialect


(lint/suspicious/noDuplicateObjectKeys)